diff --git a/node_modules/react-native-appsflyer/ios/PCAppsFlyer.m b/node_modules/react-native-appsflyer/ios/PCAppsFlyer.m
index 389726b..474bee7 100644
--- a/node_modules/react-native-appsflyer/ios/PCAppsFlyer.m
+++ b/node_modules/react-native-appsflyer/ios/PCAppsFlyer.m
@@ -7,7 +7,7 @@ static NSString *const TAG = @"[AppsFlyer_PurchaseConnector] ";
 
 #if __has_include(<PurchaseConnector/PurchaseConnector.h>)
 #import <PurchaseConnector/PurchaseConnector.h>
-#import <react_native_appsflyer-Swift.h>
+#import <react_native_appsflyer/react_native_appsflyer-Swift.h>
 
 @implementation PCAppsFlyer
 @synthesize bridge = _bridge;
